docs: tweaks to intro material for Keptn landing page (#2221)
keptn/lifecycle-toolkit: docs/content/en/docs/intro/_index.md

weight: 10
---

Keptn integrates seamlessly with cloud-native deployment tools
such as ArgoCD, Flux, and Gitlab
to bring application awareness to your Kubernetes cluster.

> Keptn supplements the standard deployment tools
> with facilities to ensure that your deployment is usable.

Reviewer: This seems like a sentence that doesn't really covey anything meaningful. Shall we maybe change it to:
```suggestion
Keptn supplements the standard deployment tools
with features to help you ensure that your deployments are in
a healthy state.
```

For information about the history of the Keptn project,
see the

> [Keptn Lifecycle Toolkit is now Keptn!](https://medium.com/keptn/keptn-lifecycle-toolkit-is-now-keptn-e0812217bf46).

Reviewer: ```suggestion
[Keptn Lifecycle Toolkit is now Keptn!](https://medium.com/keptn/keptn-lifecycle-toolkit-is-now-keptn-e0812217bf46)
```

  as well as data that comes directly from your cloud provider
  such as AWS, Google, or Azure.

* Enhances the Kubernetes
  [Horizontal Pod Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
  facility.

The Keptn metrics server unifies and standardizes
access to data from various sources,
simplifying configuration and integration into a single set of metrics.

> To learn more:

Reviewer: ```suggestion
To learn more, see:
```

> * [Custom Keptn metrics](usecase_metrics)

Reviewer: doesn't this need a `#`?
```suggestion
* [Custom Keptn metrics](usecase_metrics)
```

Author: No, usecase_metrics is a separate file under what is now the intro section.  It's a very screwy arrangement that needs to be fixed but, for now, it is what it is

  as well as data that comes directly from your cloud provider
  such as AWS, Google, or Azure.

* Enhances the Kubernetes
  [Horizontal Pod Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
  facility.

The Keptn metrics server unifies and standardizes
access to data from various sources,
simplifying configuration and integration into a single set of metrics.

To learn more, see:

* [Custom Keptn metrics](usecase_metrics)

> * [Keptn Metrics](../implementing/evaluatemetrics)

Reviewer 2: these links are not working

Author: fixed

OpenTelemetry metrics and traces)
to ensure that your deployments are observable.

To learn more, see:

* [Getting started with Keptn Observability](../getting-started)

> * [Standardize observability](../intro/usecase-observability/)

Reviewer: ```suggestion
* [Standardize observability](usecase-observability/)
```

> * [Dora metrics](../implementing/dora) guide

Reviewer: ```suggestion
* [DORA metrics](../implementing/dora) guide
```

OpenTelemetry metrics and traces)
to ensure that your deployments are observable.

To learn more, see:

* [Getting started with Keptn Observability](../getting-started)

> * [Standardize observability](usecase-observability/)

Reviewer 2: this link is not working

* [DORA metrics](../implementing/dora) guide

> * [OpenTelemetry observability](../implementing/otel) guide

Reviewer 2: this link is not working

Author: It's fixed

[KeptnApp](https://lifecycle.keptn.sh/docs/yaml-crd-ref/app/) resource,
which is a single, cohesive unit that groups multiple workloads.

> To learn more:

Reviewer: ```suggestion
To learn more, see:
```

[KeptnApp](https://lifecycle.keptn.sh/docs/yaml-crd-ref/app/) resource,
which is a single, cohesive unit that groups multiple workloads.

To learn more, see:

* [Getting started with release lifecycle management](../tutorials/tasks/)

> * [Manage release lifecycle](usecase-orchestrate)

Reviewer 2: this link does not work

Author: It's fixed

* [Deployment tasks](../implementing/tasks)

> * [Evaluations](../implementing/evaluations)

Reviewer 2: this link does not work

Author: It's fixed
